<h3> Help </h3>
<h1>Querying Aspirin database</h1>
<p>
	 You can filter the results by:
	 <ol>
		<li>experiment (3H-Acetylation or Endogenous Acetylation)</li>
		<li>peptide expectancy threshold (none, 0.05, 0.5)</li>
		<li>maximum peptide length (if no value is entered, the filter has no effect)</li>
		<li>enzyme T or TG (if none value is entered, the filter has no effect)</li>
	</ol>
</p>
<br/>

<h1>About the peptide list</h1>
<p>
	The peptide list is sorted by ascending rank product. The most significant peptide appears first in the list.
	<ol>
		<li>Rank Product: score for each peptide to sort them by significance (the lowest the better). <br/> The + replicates are being sorted by descending the pep_score, while the - reps are being sorted by ascending the pep_score. This is done to get lowest (worst) rank for the peptides located in - replicates (penalty-like scoring). Each peptide found in a replicate could be found more than once, so the rank of the hit with the highest pep_score was chosen. If a peptide is not found in a replicate means that it was highly impossible so it gets by default the lowest rank in the replicate.</li>
		<li>Penalty: additional penalty applied on the rank product.<br/>Multiplied by 10 the ranks in the - replicates. This penalty is not applied if a peptide is not found in a replicate, it just gets by default the lowest rank in the replicate.</li>
	</ol>
</p>
<br/>

<h1>Getting more information about the peptide spectrum matches</h1>
<p>
	Click on each peptide sequence, to view all the matches found for this peptide, taking into account the preselected filters. The psms are sorted by peptide score, in descending order.
	<ol>
		<li>Query: a mascot search for a single MS/MS spectra against the protein database</li>
		<li>Protein accession: identifier to search through database entries</li>
		<li>Peptide score: a peptide spectrum match (PSM) is assigned a score based on the number and type of ions that are present in the spectrum </li>
		<li>Peptide expectancy: </li>
		<li>Replicate: the raw data file</li>
		<li>Modification: locations of all modified residues and actual modified residues. K(2) means that there is a modification in the amino acid K that is in the second position through the peptide.</li>
		<li>Title: spectrum identification</li>
		<li>Number of spectrums in which each ion is assigned as the most intense.</li>
		<li>Spectrum: mass spectrum labelled with fragment ions. The figure for each psm that represents the intensities over the amino acid masses through the peptide. Red peaks are the significant ones. With black peaks is the full spectrum. </li>
		<li>Matched fragment ions table: the significant transitions. The masses are shown in a combined format of: <b>theoretical mass (assigned experimental mass : assigned intensity)</b>. The 3 most intense peaks are bolded.</li>
		<li>There are multiple psms for each peptide.</li>
	</ol>
</p>
<br/>

<h1>Getting more information about the protein</h1>
<p>
	Click on a protein accession number in a PSM, to view details like species, genename, protein description and sequence, as well as the conservation of the modification through various species.
	<ol>
		<li>
			Sequence: The peptide and its modifications highlighted through the protein sequence. The peptide is highlighted with red and the modifications with blue.
		</li>
		<li>
			Multiple alignment: The acetyl amino acid modification highlighted to view if it is conserved through different species.
			<br/><br/>
			The species aligned are:
			<br/>
			<ol>
				<li>hg19 (human)</li>
				<li>rn4 (rat)</li>
				<li>oryCun2 (rabbit)</li>
				<li>mm9 (mouse)</li>
				<li>gorGor1 (gorilla)</li>
				<li>bosTau4 (cow)</li>
				<li>danRer6 (zebrafish)</li>
				<li>galGal3 (chicken)</li>
			</ol>
		</li>
	</ol>
</p>
<br/>
