<h3> Help </h3>

<p>Some help information on how to use this website, and what the results pages mean.</p>
<p><br/></p>
<p style="font-size:1.3em">Querying Aspirin database</p>
<p>
	There are two options to filter the results. One is the name of the experiment (3H-Acetylation or Endogenous Acetylation) and second the cutoff (none, 0.05, 0.5) for the peptide expectancy value.
	
</p>
<br/>
<p style="font-size:1.3em">About the peptide list</p>
<p>
	<ol>
		<li>Rank Product: score each peptide to sort them by significance (the lowest the better)</li>
		<li>Penalty: additional small penalty in the rank product</li>
	</ol>
</p>
<br/>
<p style="font-size:1.3em">Getting more information about the peptide spectrum matches</p>
<p>
	Click on each peptide, to view all the psms that apply for this peptide, taking into account the preselected filters.
	<ol>
		<li>Query: a search for a single MS/MS spectra against the protein database</li>
		<li>Protein accession: identifier to search through database entries</li>
		<li>Peptide score: a peptide spectrum match (PSM) is assigned a score based on the number and type of ions that are present in the spectrum </li>
		<li>Peptide expectancy: </li>
		<li>Replicate: the raw data file</li>
		<li>Modification: locations of all modified residues and actual modified residues. K(2) means that there is a modification in the amino acid K that is in the second position through the peptide.</li>
		<li>Title: spectrum identification</li>
		<li>Spectra: the figure for each psm that represents the intensities over the amino acid masses through the peptide. Red peaks are the significant ones. With blue peaks is the full spectrum. </li>
		<li>Assigned y-ions table: the table that contains only the significant transitions, keeping track of their index and the theis m/z and intensity values.</li>
		<li>There are multiple psms for each peptide.</li>
	</ol>
</p>
<br/>
<p style="font-size:1.3em">Getting more information about the protein</p>
<p>
	Click on a protein accession number in a PSM, to view details like species, genename, protein description and sequence, as well as the conservation of the modification through various species.
	<ol>
		<li>
			Sequence: The peptide and its modifications highlighted through the protein sequence. Red highlight for the peptide, blue for the modifications.
		</li>
		<li>
			Multiple alignment: The acetyl amino acid modification highlighted to view if it is conserved through different species.
			<br/><br/>
			The species aligned are:
			<br/>
			- hg19 (human)<br/>
			- rn4 (rat)<br/>
			- oryCun2 (rabbit)<br/>
			- mm9 (mouse)<br/>
			- gorGor1 (gorilla)<br/>
			- bosTau4 (cow)<br/>
			- danRer6 (zebrafish)<br/>
			- galGal3 (chicken)
			
		</li>
	</ol>
</p>
<br/>
