<h3> Help </h3>

<h1>Querying Aspirin database</h1>
<p>
	 You can filter the results by:<br/>
	 - experiment (3H-Acetylation or Endogenous Acetylation)<br/>
	 - peptide expectancy threshold (none, 0.05, 0.5)<br/>
	 - maximum peptide length
</p>
<br/>

<h1>About the peptide list</h1>
<p>
	<ol>
		<li>Rank Product: score each peptide to sort them by significance (the lowest the better)</li>
		<li>Penalty: additional small penalty in the rank product</li>
	</ol>
</p>
<br/>

<h1>Getting more information about the peptide spectrum matches</h1>
<p>
	Click on each peptide sequence, to view all the matches found for this peptide, taking into account the preselected filters. The psms are sorted by peptide score, in descending order.
	<ol>
		<li>Query: a search for a single MS/MS spectra against the protein database</li>
		<li>Protein accession: identifier to search through database entries</li>
		<li>Peptide score: a peptide spectrum match (PSM) is assigned a score based on the number and type of ions that are present in the spectrum </li>
		<li>Peptide expectancy: </li>
		<li>Replicate: the raw data file</li>
		<li>Modification: locations of all modified residues and actual modified residues. K(2) means that there is a modification in the amino acid K that is in the second position through the peptide.</li>
		<li>Title: spectrum identification</li>
		<li>Spectrum: mass spectrum labelled with fragment ions. The figure for each psm that represents the intensities over the amino acid masses through the peptide. Red peaks are the significant ones. With blue peaks is the full spectrum. </li>
		<li>Assigned y-ions table: the table that contains only the significant transitions, keeping track of their index and the theis m/z and intensity values.</li>
		<li>There are multiple psms for each peptide.</li>
	</ol>
</p>
<br/>

<h1>Getting more information about the protein</h1>
<p>
	Click on a protein accession number in a PSM, to view details like species, genename, protein description and sequence, as well as the conservation of the modification through various species.
	<ol>
		<li>
			Sequence: The peptide and its modifications highlighted through the protein sequence. Red highlight for the peptide, blue for the modifications.
		</li>
		<li>
			Multiple alignment: The acetyl amino acid modification highlighted to view if it is conserved through different species.
			<br/><br/>
			The species aligned are:
			<br/>
			- hg19 (human)<br/>
			- rn4 (rat)<br/>
			- oryCun2 (rabbit)<br/>
			- mm9 (mouse)<br/>
			- gorGor1 (gorilla)<br/>
			- bosTau4 (cow)<br/>
			- danRer6 (zebrafish)<br/>
			- galGal3 (chicken)
			
		</li>
	</ol>
</p>
<br/>
